src/lib/converter/converter.ts

```diff
@@ -31,102 +30,49 @@ export interface IConverterResult
 
 
 /**
- * Event callback definition for generic converter events.
```

Comment:
Yeah, these were removed because `tsc` now complains that they're not used anywhere.